01-intro-to-R.Rmd

```diff
@@ -316,9 +315,11 @@ from. The other 4 are:
   `1+4i`) and that's all we're going to say about them
 * `"raw"` that we won't discuss further
 
+A particular and very useful data type is `"factor"` which encodes categorical variables (e.g. Habitat types, Gender etc.) 
```

Comment:
Here we talk about the atomic vectors (or as you suggest data types) in the context of what the function `typeof()` can return (as listed here: https://cran.r-project.org/doc/manuals/r-release/R-lang.html#Objects). Factors are therefore not data types. They are data structures in the same way as data frames: https://cran.r-project.org/doc/manuals/r-release/R-lang.html#Special-compound-objects